src/aiortc/jitterbuffer.py

```diff
@@ -73,7 +83,7 @@ def _remove_frame(self, sequence_number: int) -> Optional[JitterFrame]:
                 # check we have prefetched enough
                 frames += 1
                 if frames >= self._prefetch:
-                    self.remove(remove)
+                    self.remove(remove)  # this might be a bit faster than smart_remove
```

Comment:
I don't understand the comment: should we use smart_remove() or not here?

Reply:
Actually we can get rid of the remove() function entirely and replace it with smart_remove(). I haven't done so already as i didn't want to brake the backward compatibility and because smart_remove() checks the dumb_mode flag with each for loop tick (that's why it might be slightly slower). The smart_remove() run with dumb_mode flag set to true behaves exactly as the remove() function. I see 3 options here, we can:
1) leave it as it is
2) replace the remove() calls with smart_remove(dumb_mode=True) calls everywhere and get rid of remove() method
3) rename smart_remove() to remove() and replace the dumb_mode flag with smart_mode flag in the current implementation -  then we will have only remove(int) calls or remove(int, smart_mode=True) calls everywhere.

I leave the final decision to you as you now better how each of the changes can impact the rest of aiortc code.